packages/account/src/Sections/Profile/PhoneVerification/otp-verification.tsx

```diff
@@ -35,23 +34,26 @@ const OTPVerification = observer(({ phone_verification_type, setOtpVerification
     //TODO: this shall be replace by BE API call when it's ready
     const { should_show_phone_number_otp } = ui;
 
+    const reInitializeGetSettings = React.useCallback(() => {
```

Comment:
@yauheni-deriv I think we need to use React.useCallback here. As you can see that in the console i added hello inside the function. When we trigger reInitializeGetSettings without React.useCall back, it will spam the function. 

![Screenshot 2024-06-14 at 11 46 22 AM](https://github.com/suisin-deriv/deriv-app/assets/103026762/6c97b771-00d8-43a1-bee4-367f505e9a3c)

while adding the React.useCallback it will initialize the function only once:

![Screenshot 2024-06-14 at 11 51 48 AM](https://github.com/suisin-deriv/deriv-app/assets/103026762/39f8ee06-df45-49ca-9392-84ec896e2a93)

Reply:
oh, alright. I see, this is because of interval..
is there anyway to handle state for timer inside component, where timer is used. so we will not need to rerender parent every interval iteration?
if no, just ignore my comment :)

Reply:
I don't think so because we have a modal for `Didn't get the code` which is in another component, we have to share this function towards `resend-code-timer` and also `Didnt get the code modal` so have to put it in the parent component 🙏